scripts/exwafs_grib.sh

```diff
@@ -96,39 +97,39 @@ echo "#####################################"
 echo " "
 set -x
 
-if test $fcsthrs -eq 0
+if test $fhr -eq 0
 then
     echo "  "
 fi
 
-#    If we are processing fcsthrss 12-30, we have the
+#    If we are processing fhrs 12-30, we have the
 #    added variable of the a  or b in the process.
-#    The other fcsthrss, the a or b  is dropped.
+#    The other fhrs, the a or b  is dropped.
 
-if test $fcsthrs -ge 12 -a $fcsthrs -le 24
+if test $fhr -ge 12 -a $fhr -le 24
 then
-    sh $USHwafs/mkwfsgbl.sh ${fcsthrs} a
+    sh $USHwafs/mkwfsgbl.sh ${fhr} a
```

Comment:
```suggestion
    $USHwafs/mkwfsgbl.sh ${fhr} a
```
EE2 requires removing the `sh`